dot/services.go

```diff
@@ -426,8 +426,8 @@ func newSyncService(cfg *Config, st *state.Service, fg sync.FinalityGadget,
 	return sync.NewService(syncCfg)
 }
 
-func createDigestHandler(st *state.Service) (*digest.Handler, error) {
-	return digest.NewHandler(st.Block, st.Epoch, st.Grandpa)
+func createDigestHandler(cfg *Config, st *state.Service) (*digest.Handler, error) {
+	return digest.NewHandler(cfg.Log.DigestLvl, st.Block, st.Epoch, st.Grandpa)
```

Comment:
Why not get a log level argument directly instead of `cfg *Config` if we only need the log level? :thinking: